### ISSUE TITLE
Link to the constructor in GetService() method comment

### ISSUE THREAD

**Issue by metalrex100** [sarulabs/dingo#24]
Sometimes when I watch at some `GetService()` method generated by container I want to check the constructor which used by this method. Not the build func from definition, but the constructor of the Service itself.

Would be nice to have comment for the `GetService()` which will contain link to the constructor of the service.
Something like:
```// This method uses package.NewService method```

What do you think about this?

**Comment by sarulabs** [comment]
I don't understand what you mean. The build function is the constructor.

For the library, it is not possible to know what is inside the build function. Only the type of the input parameters and the type of the returned values can be known as the library uses the reflect package to determine this information.

That being said, the comments could be improved to add data from the definition, as the service name, its scope or maybe even its dependencies. A `Comment` or `Description` field could also be added to the definitions and thus in the generated comments.

**Comment by metalrex100** [comment]
The thing I would like to have - the ability to get fast the details of how service built.
Currently if I have generated container with method `GetLogger`, then to know how the logger build under the hood - I need to go to the method `SafeGetLogger` and find the row `i, err := c.ctn.SafeGet("logger")`. So the `"logger"` will be the definition name I need to search to find the details. It's pretty long way.

In my project for each service I have package.NewSerivce() constructor function which is used in definition build function. You are right, that it is not possible to know what was called inside Build func.

May be you can add `Description` or `Comment` field to the definition so I could write some custom information which will be added to `GetLogger` and `SafeGetLogger` methods comments.

**Comment by sarulabs** [comment]
I released a new version `v4.1.1` with the possibility to add a description.

There is also information about the definition included in the comments.

```go
// SafeGetService retrieves the "service" object from the main scope.
//
// Manual description if defined in the definition.
//
// ---------------------------------------------
// 	name: "service"
// 	type: *models.Service
// 	scope: "main"
// 	build: struct
// 	params:
// 		- "Value": Value(string)
// 	unshared: false
// 	close: false
// ---------------------------------------------
//
// If the object can not be retrieved, it returns an error.
```

**Comment by metalrex100** [comment]
Wow, great! Will test, thank you!
